site/_posts/2022-10-27-introducing-biometric-authentication.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blog-post
+title: Introducing biometric authentication
+description: ...
+author: Don Bergal
+image: blogs/introducing-biometric-authentication/webauthn-biometrics-launch-press-release.png
+category: blog
+tags: topic-press-release
+excerpt_separator: "<!--more-->"
+---
+
+
+FusionAuth, the developer-focused authentication platform, today announces the addition of biometric passwordless login using WebAuthn to its long list of authentication features. This enables application developers to offer the convenience and security of using built-in fingerprint and facial recognition from their mobile and laptop computers, without the need for any installed apps. 
+
+<!--more-->
+
+WebAuthn is a World Wide Web Consortium (W3C) and FIDO Alliance standard for passwordless login.  A recent report ["State of Passwordless Security"](https://f.hubspotusercontent20.net/hubfs/2670073/DL%20Assets/2022-State-of-Passwordless-Security.pdf) states that 48% of respondents found integration complexity was a barrier to implementing passwordless login. The WebAuthn standard changes the landscape for passwordless by exposing built-in biometrics on modern PCs and phones to web browsers.  
+
+FusionAuth’s built-in support for WebAuthn allows application developers to quickly enable biometric login for their customers with a quick configuration change. “Developers and SaaS providers already use FusionAuth to securely authenticate millions of their end users”, said Brian Pontarelli, CTO and founder. “We just gave them an easy option to use biometrics and eliminate stolen password risks.”   
+
+With a focus on standards based customer identity and access management, FusionAuth’s primary goal is giving developers the tools they need to add modern authentication to their applications. With APIs, documentation, sample applications, and a free unlimited-use version, FusionAuth puts developers in the driver’s seat.
+
+### Availability 
+Biometrics authentication with WebAuthn will be available for general release in November 2022 for all FusionAuth Essentials and Enterprise plan developers. Contact FusionAuth technical sales for a free trial. Learn more at [fusionauth.io/biometric](https://fusionauth.io/biometric).
+
+### Office Hours
+Join FusionAuth authentication experts & engineers to see WebAuthn in action on Friday, November 4th @ 12 pm MST live on [YouTube](https://www.youtube.com/watch?v=zx0Rfg1Vgik).  
```

Comment:
It's actually MDT right now (and will be on Nov 4). I hate that we still have to deal with this.